cx-sig-charter-draft.md

```diff
@@ -0,0 +1,81 @@
+# Contributor Experience Special Interest Group Charter
+
+This charter uses the Roles and Organization Management outlined in [sig-governance](https://github.com/spinnaker/governance/blob/master/governance.md).
+
+## Scope
+
+The Contributor Experience Special Interest Group (SIG) is responsible for improving the experience of those who contribute upstream to the Spinnaker project. We do this by creating, and maintaining programs and processes that promote community health and reduce project friction while retiring those programs and processes that don't. Being conscientious of our contributor base is critical to scaling the project, growing the ecosystem, and helping the project succeed.
+
+We do this by listening - whether it’s through our roadshows to SIG meetings, surveys, data, or [GitHub issues](https://github.com/spinnaker/spinnaker/issues), we take in the feedback and turn it into our project list. We build a welcoming and inclusive community of contributors by giving them places to be heard and productive.
+
+### In scope
+
+#### Projects & Services
+
+-  In accordance with the code of conduct, establish policies, standards, and procedures for the use, moderation, and management of all public platforms officially used by the project, including but not limited to:
+  - [GitHub Management](https://github.com/spinnaker)
+  - [Slack](https://join.spinnaker.io/)
+  - [Spinnaker](https://www.youtube.com/channel/UCcxQbw8kT1-FRhFhO2QCetg) YouTube channel
+  - Zoom
+  - Google Meet
+- Set up alerting automation for GitHub events, including new contributors, new issues, and new pull requests.
+- Work with other SIGs and interested parties in the project to execute GitHub tasks where required, see [GitHub Management](https://github.com/spinnaker) for more detail
+- Own and execute events that are targeted to the Spinnaker contributor community, including:
+  - Contributor Summit(s) 
+  - [Gardening Days and hackathons](https://github.com/spinnaker-hackathon/gardening)
+  - Retrospective moderation for other SIGs upon request
+  - Other events, like other SIG face to face events, upon request and consideration
+- Strategize, build, and execute on scalable mentoring programs for all contributor levels. These may include:
+  - [Google Summer of Code](https://summerofcode.withgoogle.com/archive/)
+  - [Outreachy](https://www.outreachy.org/)
+- New mentoring programs: office hours, meet the contributors, 1:1 pairing, group mentoring, etc.
+- Help onboard new and current contributors into the culture, workflow, and CI of our contributor experience through the [contributor guide](https://spinnaker.io/community/contributing/), other related documentation, contributor summits, and programs tailored to onboarding.
+- Perform issue triage on and maintain the spinnaker/contributor-experience repository.  
```

Comment:
1. What are the planned contents of this repository? 
1. Who will need write access to it?

Reply:
Content of this repository will be automation specific to this SIG (e.g, guides, various automation specific to contribex)
I feel to start the access will just be for leads of the contribEx sig.